app/assets/javascripts/discourse/app/components/d-editor.js

```diff
@@ -38,187 +36,10 @@ import { findRawTemplate } from "discourse-common/lib/raw-templates";
 import discourseComputed, { bind } from "discourse-common/utils/decorators";
 import I18n from "discourse-i18n";
 
-function getButtonLabel(labelKey, defaultLabel) {
-  // use the Font Awesome icon if the label matches the default
-  return I18n.t(labelKey) === defaultLabel ? null : labelKey;
-}
-
 const FOUR_SPACES_INDENT = "4-spaces-indent";
 
 let _createCallbacks = [];
 
-class Toolbar {
```

Comment:
`getButtonLabel` above and the `Toolbar` class now live in `lib/composer/toolbar`